### PR TITLE
Updated the cypher guide and removed the cypher play guide

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: src/browser/documentation/sidebar-guides/cypher.tsx [hunk 1]
```diff
@@ -22,73 +22,104 @@ import React from 'react'
...
-    </ul>
+    <p>
+      <b>NOTE:</b> This guide assumes that you use an empty graph.{' '}
+    </p>
+    <ol>
+      <li>
+        Click this code block and bring it into the Editor:
+        <pre className="pre-scrollable code runnable">
+          {
+            'CREATE (ee:Person { name: "Emil", from: "Sweden", kloutScore: 99 })'
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/cypher.tsx:52]: `"CREATE (ee:Person {name: 'Emil', from: 'Sweden', kloutScore: 99})"`

File: src/browser/documentation/sidebar-guides/cypher.tsx [hunk 2]
```diff
@@ -22,73 +22,104 @@ import React from 'react'
...
+        </pre>
+        <ul>
+          <li>
+            <code>CREATE</code> creates the node.
+          </li>
+          <li>
+            <code>()</code> indicates the node.
+          </li>
+          <li>
+            <code>ee:Person</code> – <code>ee</code> is the node variable and{' '}
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/cypher.tsx:63]: Question, what does the  `{' '}` do ?

File: src/browser/documentation/sidebar-guides/cypher.tsx [hunk 3]
```diff
@@ -22,73 +22,104 @@ import React from 'react'
...
-    </ul>
+    <p className="lead">
+      <em>Find nodes</em>
+    </p>
+    <p>Now, find the node representing Emil.</p>
+    <ol>
+      <li>
+        Click this code block and bring it into the Editor:
+        <pre className="pre-scrollable code runnable">
+          MATCH (ee:Person) WHERE ee.name = "Emil" RETURN ee;
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/cypher.tsx:86]: MATCH (ee:Person) WHERE ee.name = 'Emil' RETURN ee;

Lets try to follow the Cypher recommended string quotations `'this is the recommended style'` not `"this style".`

File: src/browser/documentation/sidebar-guides/cypher.tsx [hunk 4]
```diff
@@ -22,73 +22,104 @@ import React from 'react'
...
+          <li>
+            <code>(ee:Person)</code> is a single node pattern with label{' '}
+            <code>Person</code>. It assigns matches to the variable{' '}
+            <code>ee</code>.
+          </li>
+          <li>
+            <code>WHERE</code> filters the query.
+          </li>
+          <li>
+            <code>ee.name = "Emil"</code> compares name property to the value{' '}
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/cypher.tsx:101]: `<code>ee.name = 'Emil'</code> compares name property to the value{' '}`

**Review by martin-neotech — CHANGES_REQUESTED** [review]
Thanks, just some minor comments.

File: src/browser/documentation/sidebar-guides/cypher.tsx (hunk 2 above)
Comment (renetapopova) [reply src/browser/documentation/sidebar-guides/cypher.tsx:63]: I've just checked it adds a space explicitly; see https://stackoverflow.com/questions/38547558/why-do-people-put-in-their-react-jsx. Now that I know, I am not sure if we need it in all those places. I might check all occurrences when I finish with all guides and create a separate PR to remove the unnecessary ones.

File: src/browser/documentation/sidebar-guides/cypher.tsx [hunk 5]
```diff
@@ -19,189 +19,175 @@
...
       <li>
-        <code>MATCH</code>
-        clause to describe the pattern from known Nodes to found Nodes
+        <code>MATCH</code> describes what nodes will be retrieved based upon the
+        pattern.
       </li>
       <li>
-        <code>(ee)</code>
-        starts the pattern with a Person (qualified by WHERE)
+        <code>(ee)</code> is the anchor node that will be found based upon the{' '}
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/cypher.tsx:153]: "anchor node", is this a new term? I would call it a reference to the node.

https://neo4j.com/docs/cypher-manual/4.3/syntax/variables/

**Review by martin-neotech — APPROVED** [review]
Thanks.

**Review by OskarDamkjaer — APPROVED** [review]

**Pull request merged**